tirgei/Questioner
# #162978873 A user can sign up

#### What does this PR do?
This Pull Request creates a sign up page

#### Description of Task to be completed?
The tasks for this PR is:
- Create the sign up page 
- Style the sign up page

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/questioner.git ```

CD into the project folder on your machine
```$ cd questioner ```

Check out the branch for this PR
``` $ git checkout ft-user-signup-162978873```

Open the `register.html` file on your browser

#### Pivotal Tracker story associated with this PR
[#162978873](https://www.pivotaltracker.com/story/show/162978873)

#### Screenshots
![register](https://user-images.githubusercontent.com/17829576/50696316-296dca80-1050-11e9-8516-4fd47799bb9d.png)



Mobile 
![register-mobile](https://user-images.githubusercontent.com/17829576/50696327-3094d880-1050-11e9-98d8-feef688830b7.png)
